crates/provider-http-client/src/lib.rs

```diff
@@ -948,6 +950,21 @@ mod tests {
                 Ok(())
             }
         )?;
+    }
+
+    #[tokio::test]
```

Comment:
Since this unit test is reaching out to network, it should be `ignore`d by default